pkg/app/pagewriter/error_page.go

```diff
@@ -74,7 +91,13 @@ func (e *errorPageWriter) WriteErrorPage(rw http.ResponseWriter, status int, red
 // It is expected to always render a bad gateway error.
 func (e *errorPageWriter) ProxyErrorHandler(rw http.ResponseWriter, req *http.Request, proxyErr error) {
 	logger.Errorf("Error proxying to upstream server: %v", proxyErr)
-	e.WriteErrorPage(rw, http.StatusBadGateway, "", proxyErr.Error(), "There was a problem connecting to the upstream server.")
+	e.WriteErrorPage(rw, ErrorPageOpts{
+		Status:      http.StatusBadGateway,
+		RedirectURL: "",
```

Comment:
Just to make it clearer why we are explicitly setting this to empty string?
```suggestion
		RedirectURL: "", // The user is already logged in and has hit an upstream error. Makes no sense to redirect in this case.
```